workflows/prognostic_c48_run/runtime/transformers/fv3fit.py

```diff
@@ -1,101 +1,106 @@
 import dataclasses
-from typing import Mapping, MutableMapping, Iterable, Hashable, Sequence, Tuple
+from typing import Mapping, MutableMapping, Iterable, Hashable, Sequence, Literal
 
 import xarray as xr
 import fv3fit
-from runtime.steppers.machine_learning import non_negative_sphum_mse_conserving
+from runtime.steppers.machine_learning import (
+    non_negative_sphum_mse_conserving,
+    MultiModelAdapter,
+)
 from runtime.types import State
 from runtime.names import SPHUM, TEMP
 
 __all__ = ["Config", "Adapter"]
 
 
+@dataclasses.dataclass
+class MLOutputApplier:
```

Comment:
Yeah, there is a tension between appropriately naming config parameters and having parameters that dacite is able to interpret. This is the other option I was considering, but didn't go with since it felt like each Applier class should use the same variable name for the target name parameter (e.g. `target_name`):

```
@dataclasses.dataclass
class TendencyApplier:
    tendency_update: str
```

```
@dataclasses.dataclass
class StateApplier:
    state_update: str
```
In this case we could have `output_targets: Mapping[str, Unioin[TendencyApplier, StateApplier]]`.

Reply:
Let's just leave it as is for now. I think there's room for some pro-active development (maybe a package) for polymorphic configs since dacite-alone is not doing what we want, and this pattern has been showing up elsewhere.